src/pages/profile/profile.html

```diff
@@ -21,39 +21,46 @@
 
     <ion-row justify-content-center>
       <ion-col col-12 col-md-4 col-xl-2 text-center>
-        <ion-icon name="contact"></ion-icon>
+        <ion-icon id="pic-con" name="contact"></ion-icon>
       </ion-col>
     </ion-row>
 
-    <ion-row justify-content-center>
-      <ion-col class='name' col-12 col-md-4 col-xl-2 text-center>
+    <ion-row class="section"justify-content-center>
+      <ion-col class='name' col-12 text-center>
         Peter Horton
       </ion-col>    
     </ion-row>
     
-    <ion-row justify-content-center>
-      <ion-col col-5 align-self-center>
+    <ion-row class="section" justify-content-center>
+      <ion-col  class="user-info-colum" align-self-center>
         <ion-list>
           <ion-item>
-            <ion-label color="dark">Name</ion-label>
-            <ion-input placeholder="Peter Horton" text-center></ion-input>
-          </ion-item>
+           
+            <ion-label class= "label"color="dark">Name</ion-label>
```

Comment:
@awstin17 did some work with fontawesome icons, perhaps we could use that styling here as well?